qiskit/compiler/transpile.py

```diff
@@ -283,7 +328,7 @@ def _transpile_circuit(circuit_config_tuple: Tuple[QuantumCircuit, Dict]) -> Qua
         # Workaround for ion trap support: If basis gates includes
         # Mølmer-Sørensen (rxx) and the circuit includes gates outside the basis,
         # first unroll to u3, cx, then run MSBasisDecomposer to target basis.
-        basic_insts = ['measure', 'reset', 'barrier', 'snapshot']
+        basic_insts = ['measure', 'reset', 'barrier', 'snapshot', 'delay']
```

Comment:
I think 'delay' is required here. This is *not* for assuring 'delay' in `basis_gates` when scheduling. It's only about the workaround for ion trap support. If we removed 'delay' here, when transpiling circuits with delays and rxxs, unnecessary ['u3', 'cx'] would be added to `basis_gates`. I'm not sure how harmful such unnecessary basis gates are in the latter process (they might affect nothing), but it would be better not to remove 'delay' here.

Reply:
Ok thank you, @itoko :)